src/main/java/at/hannibal2/skyhanni/utils/LorenzUtils.kt

```diff
@@ -69,9 +68,6 @@ object LorenzUtils {
 
     val debug: Boolean get() = onHypixel && SkyHanniMod.feature.dev.debug.enabled
 
-    // TODO move into lorenz logger. then rewrite lorenz logger and use something different entirely
```

Comment:
please revert this change here. not rlevant to 1.21 and causes merge conflicts

Reply:
ok but im keeping it in the other place for now